src/core/InputDropdown/style.ts

```diff
@@ -288,7 +276,6 @@ export const StyledInputDropdown = styled(Button, {
 export const StyledDetail = styled("span", {
   shouldForwardProp: (prop: string) => !doNotForwardProps.includes(prop),
 })`
-  ${labelFontBodyXs}
```

Comment:
Here's where we're removing the original override for all InputDropdowns to be size xs (now xs is the default which they inherit, so it doesn't need to be set here)